src/js/components/stock-movement-wizard/StockMovementCombinedShipments.jsx

```diff
@@ -0,0 +1,235 @@
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import PropTypes from 'prop-types';
+import { getTranslate } from 'react-localize-redux';
+
+import CreateStockMovement from './combined-shipments/CreateStockMovement';
+import AddItemsPage from './combined-shipments/AddItemsPage';
+import SendMovementPage from './combined-shipments/SendMovementPage';
+import Wizard from '../wizard/Wizard';
+import apiClient from '../../utils/apiClient';
+import { showSpinner, hideSpinner, fetchTranslations, updateBreadcrumbs, fetchBreadcrumbsConfig } from '../../actions';
+import { translateWithDefaultMessage } from '../../utils/Translate';
+
+import './StockMovement.scss';
+
+/** Main combined shipments stock movement form's wizard component. */
+class StockMovements extends Component {
```

Comment:
This should be named the same as file name to distinguish this from StockMovement.